miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/UserInfoBridgeWrapper.kt

```diff
@@ -0,0 +1,147 @@
+package com.rakuten.tech.mobile.miniapp.js
```

Comment:
This file can be moved to `com.rakuten.tech.mobile.miniapp.js.userinfo`.

Reply:
Let me move it.